### PR TITLE
YEP - Updated Badwords and Links

### PR DESCRIPTION
**Pull request by jani270**
Hi, in this change ill add 2 Scamlinks (normal) and these 2 other words. I think adding them and enforcing our NSFW Rule would make Moulberry Bush and the general chat a better place. In the last few weeks the chat there is most of the time very unreadable and contains just the words "sex", "horny", "fuck me" or smth else. 

If you dont want these changes just remove them, but please iron think about them.

### CODE REVIEW

**Comment by IRONM00N** [comment]
ill have to refactor this but I need to think about it

**Comment by jani270** [comment]
> ill have to refactor this but I need to think about it

Oky maybe a discussion in staff chat would be also good idk how other staffs see that

**Comment by IRONM00N** [comment]
I'm not sure automod is the best way to address this, it has no awareness to context

File: lib/badwords.json5 [hunk 1]
```diff
@@ -38,5 +38,9 @@
 	"hi bro, i'm leaving this fucking game, take my skin": 3,
 	'hello i am leaving cs:go': 3,
 	'hi, bro i am leaving cs:go and giving away my skin': 3,
-	'tar baby': 2
+	'tar baby': 2,
+	'cum': 0,
+	'sex': 0,
```
Comment (IRONM00N) [inline lib/badwords.json5:43]:
```suggestion

```

File: lib/badwords.json5 [hunk 2]
```diff
@@ -38,5 +38,9 @@
 	"hi bro, i'm leaving this fucking game, take my skin": 3,
 	'hello i am leaving cs:go': 3,
 	'hi, bro i am leaving cs:go and giving away my skin': 3,
-	'tar baby': 2
+	'tar baby': 2,
+	'cum': 0,
```
Comment (IRONM00N) [inline lib/badwords.json5:42]:
```suggestion
```

File: lib/badwords.json5 [hunk 3]
```diff
@@ -38,5 +38,9 @@
...
+	'sex': 0,
+	'seggs': 0,
```
Comment (IRONM00N) [inline lib/badwords.json5:44]:
```suggestion
```

File: lib/badwords.json5 [hunk 4]
```diff
@@ -38,5 +38,9 @@
...
+	'cock': 0,
```
Comment (IRONM00N) [inline lib/badwords.json5:45]:
```suggestion
```

**Review by IRONM00N — CHANGES_REQUESTED** [review]
I am going to be implementing these a different way

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**